packages/vc-http-api-test-server/services/sanitizeAllResults.js

```diff
@@ -0,0 +1,32 @@
+module.exports = extractTestSummary = (results) => {
```

Comment:
this structure looks valuable to did core, please open an issue on that suite to track alignment with changes here.